vumi_twilio_api/twilio_api.py

```diff
@@ -62,10 +79,28 @@ def format_xml(dct, root=None):
 
     @staticmethod
     def format_json(dct):
-        return json.dumps(dct)
+        if dct.get('Call'):
+            dct = dct['Call']
+        if dct.get('Version'):
+            dct = dct['Version']
```

Comment:
Let's keep the root resource -- seems sane to have it. Maybe let's document
the reason for it with a comment? Or add a flag to the method that the root
resource can add to get the custom behaviour it needs?